Tools/Scripts/filter-test-logs

```diff
@@ -53,10 +53,9 @@ def filter_jsc_tests(log_file_name):
             elif line.startswith('** The following JSC') or line.startswith('Results'):
                 print_line = True
                 print(line, end='')
+    return print_line
```

Comment:
we should rename `print_line` variable name to something which indicates the intent more clearly, maybe `found_test_completion_results` . Although that's a matter of a separate PR. cc @briannafan

Reply:
Yah. I don't think there's a good name for a boolean that happens to represent two different things (whether to print a line and whether we've seen results. Perhaps we could accumulate the results and have the caller handle the printing (or error out if there are none). As you say, something for a follow-up PR.